package.json

```diff
@@ -25,6 +25,7 @@
     "seed": "ts-node ./seed/seed.ts"
   },
   "dependencies": {
+    "@aws-sdk/client-s3": "^3.701.0",
```

Comment:
you can uninstall this since you've taken out the S3 stuff and we can add it back whenever we do that functionality